sale_order_line_position/models/position_mixin.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class OrderPositionMixin(models.AbstractModel):
+    _name = "order.position.mixin"
+    _description = "Order position mixin"
+
+    locked_positions = fields.Boolean(compute="_compute_locked_positions")
+
+    @api.depends("state")
```

Comment:
```suggestion
    @api.depends(self._compute_locked_positions_depends())
```
and let the consumer of the mixin pass the fields